concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1144,6 +1144,34 @@ jobs:
 
                   cf push
 
+        - task: sync-admin-users
+          config:
+            platform: linux
+            image: docker:///ruby#2.2
```

Comment:
It would be preferable not to run `bundle install` within the pipeline. I think it'd also be good to make the dependency more explicit. What do either of you think about adding it (and a test for the binary) to the `cf-uaac` container?